requirements.txt

```diff
@@ -1,6 +1,7 @@
 Flask
 Flask-SQLAlchemy
 Flask-WTF
+gunicorn
 pytest
 python-decouple
 requests
```

Comment:
This needs `psycopg2-binary` to be added back for anything that touches the db to run